Gemfile.lock

```diff
@@ -748,9 +748,6 @@ GEM
       addressable (>= 2.3.6)
       crack (>= 0.3.2)
       hashdiff
-    websocket-driver (0.7.0)
-      websocket-extensions (>= 0.1.0)
-    websocket-extensions (0.1.3)
```

Comment:
:warning: I saw this on my local machine as well, but I don't know when. Worth checking.

Reply:
ok, it came with the headless chrome branch (probably an error) and was removed in the recent paper_trail upgrade PR (just because it run the bundle install). I'll rebase this PR and this will disappear.